src/vit-servicing-station/vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs

```diff
@@ -949,4 +951,27 @@ mod test {
 
         assert_eq!(tags, vec!["tag_a", "tag_b"]);
     }
+
+    #[tokio::test]
+    async fn test_put_raw_snapshot() {
+        let content = r#"{"snapshot":[{"stake_public_key":"0xa6a3c0447aeb9cc54cf6422ba32b294e5e1c3ef6d782f2acff4a70694c4d1663","voting_power":2,"reward_address":"0xa6a3c0447aeb9cc54cf6422ba32b294e5e1c3ef6d782f2acff4a70694c4d1663","delegations":"0x0000000000000000000000000000000000000000000000000000000000000000","voting_purpose":0},{"stake_public_key":"0x00588e8e1d18cba576a4d35758069fe94e53f638b6faf7c07b8abd2bc5c5cdee","voting_power":1,"reward_address":"0x00588e8e1d18cba576a4d35758069fe94e53f638b6faf7c07b8abd2bc5c5cdee","delegations":"0x0000000000000000000000000000000000000000000000000000000000000000","voting_purpose":0}],"update_timestamp":"1970-01-01T00:00:00Z","min_stake_threshold":0,"voting_power_cap": "NaN","direct_voters_group":null,"representatives_group":null}"#;
```

Comment:
what happens when voting power cap is 0.56 or some other float?

Reply:
More detailed explanation about the `voting_power_cap` and why it is needed you can find here https://github.com/input-output-hk/catalyst-core/blob/b8690b3a73f291063ccf5c9066afb0b26eebd49b/src/catalyst-toolbox/snapshot-lib/src/influence_cap.rs#L31-L32 .
The exact values as 0.56 or 1.0 I just established as an examples inside the error description. Actually it can be any value